src/domain/collaboration/callout/CalloutsInContext/CalloutsGroupView.tsx

```diff
@@ -13,6 +13,7 @@ interface CalloutsGroupProps extends CalloutsViewProps {
   groupName: CalloutGroupName;
   flowState?: string;
   createButtonPlace?: 'top' | 'bottom';
+  hideButton?: boolean;
```

Comment:
can we call this `hideCreateButton`? 
I would even try to use `createButtonPlace` if it's clean. something like `'top' | 'bottom' | 'none'`